docs/src/Run/AccountService/RunAccountService.md

```diff
@@ -0,0 +1,49 @@
+<!-- TODO: Update for Builder Track, see: https://github.com/ProjectLibertyLabs/gateway/issues/630 -->
+# Running Account Service
+
+## Prerequisites
+
+In order to run this project you need:
+
+- [Node.js](https://nodejs.org)
+- [Docker](https://docs.docker.com/get-docker/)
+
+## Environment Variables
+
+Modify any environment variables in the `.env` file as needed. The complete set of environment variables is documented [here](./ENVIRONMENT.md), and a sample environment file is provided [here](../../env-files/account.template.env).
+
+1. Copy the template values into the .env.account file.
+
+  ```sh
+     cp env-files/account.template.env .env.account
+  ```
+
+2. Replace template values with values appropriate to your environment.
+
+## Install
+
+Install NPM Dependencies:
+
+```sh
+  npm install
+```
+
+## Usage
+
+Note: using [docker compose file](../../docker-compose.yaml) to start the services. This will start the services in development mode.
+
```

Comment:
Recommend replacing with:
"Note: Using [docker compose file](../../docker-compose.yaml) to start the services will start the services in development mode."